CODEOWNERS

```diff
@@ -1,15 +1,15 @@
 # catch-all
 
-* @input-output-hk/cardano-sl
+* @input-output-hk/core-tech
```

Comment:
Check with @disassembler about this. I think he purposely had cardano-sl as a catch all

Reply:
We don't have that group anymore, also it was renamed: https://input-output-rnd.slack.com/archives/C05CQ9Y3JMC/p1686923478805939
And there's this PR i just noticed https://github.com/input-output-hk/cardano-node/pull/5346